.github/workflows/continous-integration.yml

```diff
@@ -973,6 +973,7 @@ jobs:
 
     # deploy will only be run when there is a tag available
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && github.repository == 'RasaHQ/rasa'
+    needs: [wait_for_docs_tests, docker] # only run after all other stages succeeded
```

Comment:
Do we need `wait_for_docs_tests`?

Reply:
Not sure, i didnot skip this because it had a comment
`    # Looks for doc test workflows and waits for it to complete successfully
    # Runs on tag pushes and pushes to main exclusively, as it is a dependency of release jobs`

Reply:
I'd check with @m-vdb if this is really required 🤔

Reply:
if you're skipping the doc tests on the tag push, you can skip the wait_for_docs_tests job

Reply:
(and edit the comment to reflect the current state :) )

Reply:
yes I understand that. I am saying that it's unnecessary to run the docs tests on the tag push. The same reason why it's unnecessary to run the tests on the tag push.

Reply:
gotcha! Also skipped doc tests workflow on tag push this time.